package.json

```diff
@@ -65,18 +77,18 @@
 		"setup": "npm run identify && npm run build-python skip",
 		"eject": "react-scripts eject",
 		"predev": "npm run setup",
-		"dev": "cross-env REACT_APP_DEPLOYMENT=dev REACT_APP_TARGET=demo react-app-rewired start",
+		"dev": "cross-env VITE_REACT_APP_DEPLOYMENT=dev VITE_REACT_APP_TARGET=demo vite",
 		"prestart": "npm run setup skip",
-		"start": "react-app-rewired start",
-		"start-demo": "cross-env REACT_APP_TARGET=demo npm run start",
-		"start-plg": "cross-env REACT_APP_ALT_AUTH=plg npm run start",
+		"start": "vite serve",
+		"start-demo": "cross-env VITE_REACT_APP_TARGET=demo npm run start",
+		"start-plg": "cross-env VITE_REACT_APP_ALT_AUTH=plg npm run start",
 		"test": "react-app-rewired test",
```

Comment:
do we need `react-app-rewired` for tests ?

Reply:
tests are moved to vitest